packages/tlstore/src/lib/StoreSchema.ts

```diff
@@ -51,8 +51,12 @@ export type StoreSchemaOptions<R extends BaseRecord, P> = {
 	createIntegrityChecker?: (store: Store<R, P>) => void
 	/** @internal */
 	derivePresenceState?: (store: Store<R, P>) => Signal<R | null>
+	createProjectName?: () => string
 }
 
+/** @internal */
+export const DEFAULT_PROJECT_NAME = 'Project'
+
```

Comment:
I think this should live in the tlschema package, and we should have a way to provide a custom project name that doesn't involve adding the `createProjectName` stuff. Maybe you could add it as one of the store props in TLStore.ts?

https://github.com/tldraw/tldraw/blob/fff9c4c09b410a41a84a252eaaab64626a1127b3/packages/tlschema/src/TLStore.ts#L56-L62

Reply:
Cool, will give this a go!